src/ray/gcs/test/gcs_test_util.h

```diff
@@ -83,6 +83,26 @@ struct Mocker {
     return request;
   }
 
+  static std::vector<std::shared_ptr<const BundleSpecification>> GenBundlesSpecification(
```

Comment:
```suggestion
  static std::vector<std::shared_ptr<const BundleSpecification>> GenBundleSpecifications(
```
?

Reply:
This is not addressed?

Reply:
So sorry about this... I didn't mean to do this, I think I didn't notice the `s` at that time.....

Reply:
No worries haha. I just wanted to make sure things are not resolved without comment! (since that means I need to review more carefully second time lol. Usually I assume that my initial comments are already addressed)

Reply:
updated